content/applications/hr/employees/new_employee.rst

```diff
@@ -0,0 +1,259 @@
+     :align: center
+     :alt: Add the work information to this section.
+
+Private information tab
+-----------------------
+
+No information in the :guilabel:`Private Information` tab is required, however, some information in
+this section may be critical for the company's payroll department. In order to properly process
+payslips and ensure all deductions are accounted for, the employee's personal information should be
+entered.
+
+Here, the employee's :guilabel:`Private Contact`, :guilabel:`Marital Status`, :guilabel:`Emergency
+Contact`, :guilabel:`Education`, :guilabel:`Citizenship`, :guilabel:`Dependant`, and
+:guilabel:`Work Permit` information is entered. Fields are entered either using a drop-down menu,
+clicking a check box, or typing in the information.
+
+- :guilabel:`Private Contact`: Enter the :guilabel:`Address` for the employee. The selection can be
+  made with the drop-down menu. If the information is not available, type in the name for the new
+  address. To edit the new address, click the :guilabel:`External Link` icon to open the address
+  form. On the address form, enter the necessary details, then click :guilabel:`Save`. Some other
+  information in the :guilabel:`Private Contact` section may auto-populate, if the address is
+  already listed in the drop-down menu.
+- :guilabel:`Marital Status`: Select either :guilabel:`Single`, :guilabel:`Married`,
+  :guilabel:`Legal Cohabitant`, :guilabel:`Widower`, or :guilabel:`Divorced` from the drop-down
+  menu.
+- :guilabel:`Emergency Contact`: Type in the name and phone number of the employee's emergency
+  contact.
+- :guilabel:`Education`: Select the highest level of education completed by the employee from the
+  drop-down menu. Options include :guilabel:`Graduate`, :guilabel:`Bachelor`, :guilabel:`Master`,
+  :guilabel:`Doctor`, or :guilabel:`Other`. Type in the :guilabel:`Field of Study`, and the name of
+  the :guilabel:`School` in the respective fields.
+- :guilabel:`Citizenship`: This section houses all the information relevant to the citizenship of
+  the employee. Some selections use a drop-down menu, as does the :guilabel:`Nationality (Country)`,
+  :guilabel:`Gender`, and :guilabel:`Country of Birth` sections. The :guilabel:`Date of Birth` uses
+  a calendar module to select the date. First, click on the name of the month, then the year, to
+  access the year ranges. Use the left and right arrows, navigate to the correct year range, and
+  click on the year. Next, click on the month. Last, click on the day to select the date. Type in
+  the information for the :guilabel:`Identification No`, :guilabel:`Passport No`, and
+  :guilabel:`Place of Birth` fields. If the employee is :guilabel:`Disabled` or a
+  :guilabel:`Nonresident`, click the check box next to the respective fields.
+- :guilabel:`Dependant`: If the employee has any dependants, that information is entered here. Type
+  in the :guilabel:`Number of Children` the employee has, and check the boxes next to
```

Comment:
This is an instruction to type in the _number of children_ so a :guilabel: is unnecessary here
```suggestion
  in the number of children the employee has, and check the boxes next to
```